### PR TITLE
[gui-tests][full-ci] fix gui tests

### PR DESCRIPTION
**Pull request by saw-jan**
Test adjustment according to change in source code: 

1. updated selector for context menu
2. updated screenshot verification point
3. skipped tests related to sharing via public link with expiration date

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -24,8 +24,8 @@ OC_UBUNTU = "owncloud/ubuntu:20.04"
 # Eventually, we have to use image built on ubuntu
 # Todo: update or remove the following images
 # https://github.com/owncloud/client/issues/10070
-OC_CI_CLIENT_FEDORA = "owncloudci/client:fedora-38-amd64"
-OC_CI_SQUISH = "owncloudci/squish:fedora-38-7.1.1-qt64x-linux64"
+OC_CI_CLIENT_FEDORA = "sawjan/client:qtlib"
+OC_CI_SQUISH = "sawjan/squish:qtlib"
```
Comment (saw-jan) [inline .drone.star:28]: could you revert these now? the updated images should be available

Comment (saw-jan) [inline .drone.star:28]: @Salipa-Gurung could you revert these now? the updated images should be available

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by Salipa-Gurung — APPROVED** [review]
:+1:

**Pull request merged**